arviz/plots/posteriorplot.py

```diff
@@ -117,17 +117,25 @@ def plot_posterior(
     backend: str, optional
         Select plotting backend {"matplotlib","bokeh"}. Default "matplotlib".
     backend_kwargs: bool, optional
-        These are kwargs specific to the backend being used. For additional documentation
-        check the plotting method of the backend.
+        These are kwargs specific to the backend being used, passed to
+
```

Comment:
There seems to be an extra blank line here.

Reply:
In rST (the markup language used in docstrings) breaklines are ignored, consecutive lines are considered a single line, rendered to html and then the actual breaklines are decided according to the page length, so the texts always looks somewhat good. To _force_ a breakline and split content between paragraphs, a blank line is left between the pieces of text to be separated. 

Therefore, having this space forces a breakline right after the `passed to` which might not be the right place to break depending on the screen. 

On my computer, the docs look like this:

![image](https://user-images.githubusercontent.com/23738400/136225260-39aaaf2e-4eae-4f9b-ae56-faede8f784a2.png)

so it seems it won't make a difference for my screen size, but on a tabled they'd look like this:

![image](https://user-images.githubusercontent.com/23738400/136225056-d3283ed0-6d52-425a-818b-ee1d15bc1f23.png)

instead of having the subplots link on the above line.

Reply:
I fixed this